.env.example

```diff
@@ -73,5 +73,5 @@ ENABLE_WW1_FULLTEXT_SEARCH=0
 DISQUS_SHORTNAME=shortname-here
 
 # Enable authentication for User galleries (XX), enabled by default
-# portal must be running on https (port 3001 for localhost)
 ENABLE_XX_USER_AUTH=1
+OAUTH_URL=https://keycloak-test.eanadev.org/auth/realms/europeana/protocol/openid-connect
```

Comment:
While authentication is still in development and optional, this should be commented out.